src/Calculator/Views/TitleBar.xaml.cpp

```diff
@@ -79,7 +86,8 @@ namespace CalculatorApp
 
     void TitleBar::SetTitleBarVisibility()
     {
-        this->LayoutRoot->Visibility = m_coreTitleBar->IsVisible && !m_accessibilitySettings->HighContrast ? ::Visibility::Visible : ::Visibility::Collapsed;
+        // this->BackgroundElement->Visibility = m_coreTitleBar->IsVisible && !m_accessibilitySettings->HighContrast ? ::Visibility::Visible : ::Visibility::Collapsed;
+        this->LayoutRoot->Visibility = m_coreTitleBar->IsVisible ? ::Visibility::Visible : ::Visibility::Collapsed;
```

Comment:
Warning, this diff will change how the title bar looks when high-contrast is on (different background color), is it intended?

Reply:
Correct.